rfcs/xxxx-v5-database.md

```diff
@@ -0,0 +1,169 @@
+
+While the probability of two UUIDs colliding is not zero, it is practically negligible, making them an excellent choice for unique identifiers.
+
+## Links
+
+`Links` introduce the following key functionalities:
+
+### Relate entry to document
+
+In V4, you might have encountered this situation:
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/01061116-478b-465f-9b3b-a88ea02598c6" width="600px" />
+</p>
+
+
+When relating an article to a specific author in V4, only articles and authors of the same locale could be linked, because localizations were not really grouped in a single document.
+
+Thus, if an author did not have a locale that matched the article's, creating a relationship was impossible.
+
+`Links` will solve this by referencing a document as a whole, not a specific locale:
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/da52c5de-57c2-4ced-8986-e0f6787d8892" width="600px" />
+</p>
+
+
+### Synchronize links across locales
+
+In V4, if you want to keep the same relation across the same locales, one would have to manually create them, with the risk of missing one.
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/0e2729fe-7673-490b-9493-9ea3a8fb4470" width="600px" />
+</p>
+
+In V5, `links` will offer the possibility to be synchronized across locales.
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/0d5841dd-89b1-42f6-bed5-b463f40f515b" width="600px" />
+</p>
+
+As an additional comment, the synched relation will only affect the locales inside the same publication state. Meaning, relations will not be shared between the `draft` and `published` versions of the document.
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/4a69c2e3-628e-45b1-8c77-338272a61c32" width="600px" />
+</p>
+
+
+### Polymorphic links
+
+V5 will eventually support polymorphic links.
+
+Those will allow use cases like Website Menus , linked to many other content types
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/29628824-c735-4776-b651-4c99ad11db8f" width="500px" />
+</p>
+
+This was just not possible in V4.
+
+## Link content schema
+
+Links will be represented as so in the content type schema:
+
+```jsx
+{
+  attributes: {
+    myLink: {
+      type: "link",
+      // Target one (or multiple) content types.
+      // Target all by not defining target
+      target: "uid-a" | ["uid-a", "uid-b"] | undefined,
+      multiple: true|false
+    }
+  }
+}
+```
+
+## Links Database Schema
+
+The database schema of `links` join tables :
+
+<p align="center">
+	<img src="https://github.com/strapi/rfcs/assets/20578351/f6ebb23e-c6e7-47b6-bdae-8030c80786a6" width="700px" />
```

Comment:
pivot_publicationState should be pivot_publication_state. And i think the names should not be prefixed.